PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/CustomerSurveyHelper.java

```diff
@@ -72,8 +71,6 @@ public void showFeedbackNotification(Project project) {
                     .subscribeOn(new IdeaSchedulers(project).dispatchUIThread())
                     .take(1)
                     .subscribe(next -> {
-                        SurveyPopUpDialog dialog = new SurveyPopUpDialog(CustomerSurveyHelper.this, project);
-                        dialog.setVisible(true);
                         synchronized (CustomerSurveyHelper.class) {
```

Comment:
lock class is not a good idea. although it is not changed by you in this PR.

Reply:
This is the outdated codes which has been removed in next pr